packages/data/src/registry.js

```diff
@@ -214,6 +223,7 @@ export function createRegistry( storeConfigs = {}, parent = null ) {
 	}
 
 	let registry = {
+		parent,
```

Comment:
Why do we need parent in the API of the registry?

Reply:
To be able to traverse up the registries tree to find the correct registry which registered the store.
See https://github.com/WordPress/gutenberg/pull/26724/files#diff-842c1e34fc3a1df56dc553d99766a9f86a3e0e141a99632471ed462e2b04035fR39-R54

Reply:
I wonder if there's a better alternative? I dislike that we have to add additional API to `registry`, but it's the easiest way I can think of for now.

Reply:
In my mind even `stores` should be private (though it's not new).;

I think what you really want is a function to subscribe to just a given store so maybe `registry.__experimentalSubscribeStore( storeName, handler )` 

or potentially  update `registry.subscribe` to allow  `registry.subscribe( storeName, handler )`

Reply:
Cool, cool! I pushed 9295f6d to use a `WeakMap` instead to store these values privately (without breaking changes of course).

Reply:
The solution isn't perfect, have to do some workarounds here and there due to the legacy `withPlugins` and `use` methods.

An alternative solution is to use an `_id` and store it in the registry object, the downside obviously is that we're now introducing a private API to the registry object though.